pkg/agent/templates/templates_windows.go

```diff
@@ -165,6 +168,15 @@ func ParseTemplateFromConfig(templateBuffer string, config interface{}) (string,
 		"replace": func(s string) string {
 			return strings.ReplaceAll(s, "\\", "\\\\")
 		},
+		"deschemify": func(s string) string {
+			logrus.Infof("The current address is %s", s)
+			if strings.HasPrefix(s, "npipe:") {
+				u, _ := url.Parse(s)
```

Comment:
What could happen if there's an error but we ignore it here?

Reply:
I will handle that by returning an empty string.